mithril-client-cli/src/utils/unpacker.rs

```diff
@@ -36,7 +36,7 @@ pub enum SnapshotUnpackerError {
     UnpackDirectoryAlreadyExists(PathBuf),
 
     /// Cannot write in the given directory.
-    #[error("Unpack directory '{0}' is not writable.")]
+    #[error("Unpack directory '{0}' is not writable, please check own or parents' permissions and ownership.")]
```

Comment:
```suggestion
    #[error("Unpack directory '{0}' is not writable, please check own or parent's permissions and ownership.")]
```

Reply:
It was not a typo as a directory could be unwritable also because of permission issues in its parent's parent, I think

Reply:
My bad sorry 👍